README.md

```diff
@@ -100,3 +101,6 @@ For more information, please refer to the [official documentation][docs-url].
 
 [gitter-url]: //gitter.im/automotivemastermind/condo
 [gitter-image]:https://img.shields.io/badge/⊪%20gitter-join%20chat%20→-1dce73.svg
+
+[issues-url]: //waffle.io/automotivemastermind/condo
+[issues-image]: https://badge.waffle.io/automotivemastermind/condo.svg?label=backlog&title=Backlog
```

Comment:
Please don't include the scheme in URIs, so it matches the scheme used by the caller (part HTTPS causes a warning in some browsers)